### PR TITLE
translate 'set up Konnectivity service' into chinese

### PR DESCRIPTION
**Pull request by fancc**
translate 'set up Konnectivity service' into chinese

### CODE REVIEW

**Comment by sftim** [comment]
Please rebase this pull request against _master_ to bring in a work-around for issue #22222.

**Comment by tengqm** [comment]
@fancc pls rebase this onto master?

**Comment by fancc** [comment]
> @fancc pls rebase this onto master?

done

**Comment by tengqm** [comment]
you need to copy four YAML files to zh/examples directory into the correct path.

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Deploy the Konnectivity server on your control plane node. The provided
+`konnectivity-server.yaml` manifest assumes
+that the Kubernetes components are deployed as a {{< glossary_tooltip text="static Pod"
+term_id="static-pod" >}} in your cluster. If not, you can deploy the Konnectivity
+server as a DaemonSet.
+-->
+接下来，你需要部署 Konnectivity 服务器和代理。[kubernetes-sigs/apiserver-network-proxy](https://github.com/kubernetes-sigs/apiserver-network-proxy) 是参考实现。
+
+在控制平面节点上部署 Konnectivity 服务，下面提供的 `konnectivity-server.yaml` 配置清单假定您在集群中将 Kubernetes 组件都是部署为{{< glossary_tooltip text="static Pod" term_id="static-pod" >}}。如果不是，你可以将 Konnectivity 服务部署为 DaemonSet。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:57]: - 注意换行
- tooltip 中的 text 要翻译

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+content_type: task
+weight: 70
+---
+
+<!-- overview -->
+<!--
+The Konnectivity service provides a TCP level proxy for the control plane to cluster
+communication.
+-->
+Konnectivity 服务为控制平面提供集群通信的TCP级别代理。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:12]:
```suggestion
Konnectivity 服务为控制平面提供集群通信的 TCP 级别代理。
```

**Pull request merged**